build.sbt

```diff
@@ -1,21 +1,21 @@
 import ReleaseTransformations._
 
 lazy val baseSettings = Seq(
-  scalaVersion := "2.12.18",
+  scalaVersion := "2.13.11",
```

Comment:
Why did you not upgrade to 2.13.12? Is there a reason for that, like compile errors?

Reply:
Ah, we recently got slightly burned on a different project by a dependency that failed on Scala 2.13.12 (see https://github.com/scala/bug/issues/12862 & https://github.com/scala/bug/issues/12862#issuecomment-1799265691)... we're not using that dependency on _this_ project, and I could have upgraded to 2.13.12 - I just needed a bit of space to breath and not handle another complexity right now!